### DISCUSSION TITLE
Have you metadata for the GEMs from gut

### DISCUSSION THREAD

**Discussion by hongzhonglu** [franciscozorrilla/thesis_pipeline#78] [Q&A] [answered]
Dear Francisco,
Nice toolbox. Thanks for your effort. As I am interested in the gut GEMs, I downloaded GEMs from your article generated by metaGEM. Do you have metadata information for these gut GEMs, like the personal information for each gut GEMs?

Best,
Hongzhong

**Comment by franciscozorrilla** [comment, marked as answer]
Dear Hongzhong,

Unfortunately I do not have a nice file with all this information to share with you yet, but I can tell you how you can get it. This is the [article](https://www.nature.com/articles/nature12198) where the samples were taken from, and here is the download [link](https://static-content.springer.com/esm/art%3A10.1038%2Fnature12198/MediaObjects/41586_2013_BFnature12198_MOESM507_ESM.xlsx) for the supplementary data. The first sheet (named "Supplementary Table 3") has the metadata for the patient samples. Here is a [txt file](https://github.com/franciscozorrilla/metaGEM/files/6068113/metadata.txt) connecting the sample filenames (e.g. ERR260137) with the patient sample ID (e.g. 330). Each GEM should be named according to {sample_ID}_{mag_ID}.xml, so you should now be able to connect each GEM with its patient metadata. Please let me know if you have any further questions. Hope it helps and thanks for your interest!

Best,
Francisco

**Comment by hongzhonglu** [comment]
Thanks a lot! I will check the source data file.

Best,
Hongzhong

**Comment by franciscozorrilla** [comment]
No problem! I hope to provide most of this info in the upcoming publication, but until then let me know if you are interested in any other info about the gut GEMs such as abundances, taxonomy, etc.

Best,
Francisco
